docs/concepts/resource-manager.md

```diff
@@ -109,6 +109,7 @@ The gardener-resource-manager can manage a resource in different modes. The supp
 - `Ignore`
     - The corresponding resource is removed from the ManagedResource status (`.status.resources`). No action is performed on the cluster - the resource is no longer "managed" (updated or deleted).
     - The primary use case is a migration of a resource from one ManagedResource to another one.
+    - If this annotation is present on the ManagedResource itself then it will be skipped entirely by the controller (no reconciliations or deletions at all). However, when the ManagedResource itself is deleted (for example when a shoot is deleted) then the annotation is ignored and all resources will be deleted as usual. This feature can be helpful to temporarily patch/change resources managed as part of such ManagedResource.
```

Comment:
I am with mixed feelings about reusing the mode annotation for excluding ManagedResources from reconciliation. The issue that describes the various modes is https://github.com/gardener/gardener/issues/4804 - for now only the `Ignore` mode is implemented. And the mode annotation for now is supposed to be set only on resources that are managed by MR.
If we don't plan to support other "modes" on MR level, then I think it would be less confusing to use the `resources.gardener.cloud/ignore=true` annotation (or similar one) on the MR for excluding from reconciliation (in https://github.com/gardener/gardener/issues/4804 it is planned to replace the existing `resources.gardener.cloud/ignore` annotation on resources with a dedicated mode). Then we will have `resources.gardener.cloud/mode` annotation that is set only for resources in MR and `resources.gardener.cloud/ignore` that is supported on MR level only. WDYT?

Reply:
Well, I did the change, but I think all this is highly confusing and I don't see a clear concept behind it. Anyways, as said - done.